labs-ios-starter.xcodeproj/project.pbxproj

```diff
@@ -238,6 +241,7 @@
 				50FDDEA625BFC8AF007941CC /* FavoritesCollectionViewController.swift in Sources */,
 				46BEB4E924C98DAF00FE0CD1 /* ProfileListViewController.swift in Sources */,
 				463AD24524CF2CDB00447BB8 /* AddProfileViewController.swift in Sources */,
+				1753D83425BFE231004F9E73 /* MyProfileViewController.swift in Sources */,
```

Comment:
I would have named it just `ProfileViewController` without the `my` but that's fine